### PR TITLE
Add support for configuring the plugin to only emit warnings

### PR DESCRIPTION
**Pull request by oyvindberg**
I wanted this when retrofitting a project with acyclic after it was written so it can be done a bit more piece by piece. WDYT?

### CODE REVIEW

**Comment by akashSri92** [comment]
Nice.. I have a requirement which needs just this.

File: build.sbt [hunk 1]
```diff
@@ -3,7 +3,7 @@ organization  := "com.lihaoyi"
 
 name := "acyclic"
 
-version := "0.1.7"
+version := "0.1.8"
```
Comment (lefou) [inline build.sbt:6]: This does not belong to the PR. Version will be updated when release is made.

**Review by lefou — COMMENTED** [review]
I like this PR and it looks good to me. Could you please rebase it on current master to resolve the conflicts. Se also my comments below.

**Comment by lefou** [comment]
I merged current master branch and resolved my own review comments. @lihaoyi WHDY?

**Comment by oyvindberg** [comment]
5 years, let's go! :)

**Comment by lefou** [comment]
Thank you!
